modules/custom/social_graphql/src/GraphQL/EntityConnection.php

```diff
@@ -182,6 +182,9 @@ public function nodes() : SyncPromise {
    * returning edges or nodes, this needs to be compensated in the same way.
    * This function removes the overfetching and ensures the results are in the
    * requested order.
+   *
+   * @return \GraphQL\Executor\Promise\Adapter\SyncPromise
+   *   The edges.
```

Comment:
```suggestion
   *   The result in the order as requested by the client which may differ from how we get it from the database in `getResult`.
```

Linebreaks may need to be added for coding standards.